Jenkinsfile_pipeline_test

```diff
@@ -0,0 +1,47 @@
+#!groovy
+
+// THIS PIPELINE IS FOR TESTING SHARED PIPELINE CODE ONLY
+
+properties([
+  parameters([
+    string(name: 'LIB_VERSION', defaultValue: 'master', description: 'Branch name of pipeline library to use')
+  ])
+])
+
+library "Infrastructure@${params.LIB_VERSION}"
+
+String product = "cmc"
+String component = "citizen-frontend"
+
+List<LinkedHashMap<String, Object>> secrets = [
```

Comment:
this enough vars for this to run?
if you want to keep this you could move it out to a separate PR?

Reply:
ill update 👍